api/src/pcapi/routes/backoffice/templates/offerer/get.html

```diff
@@ -174,7 +174,18 @@ <h5 class="modal-title">Supprimer la structure {{ offerer.name }}</h5>
               <p class="mb-1">
                 <a href="{{ offerer | pc_pro_offerer_offers_link }}"
                    target="_blank"
-                   class="fw-bold link-primary">Offres associées</a>
+                   class="fw-bold link-primary">
+                  Offres associées
+                  <i class="bi bi-box-arrow-up-right"></i>
+                </a>
+              </p>
+              <p class="mb-1">
+                <a href="{{ url_for("backoffice_web.offer.list_offers") }}?offerer={{ offerer.id }}"
```

Comment:
La bonne syntaxe est :
```suggestion
                <a href="{{ url_for("backoffice_web.offer.list_offers", offerer=offerer.id) }}"
```
Pareil pour les autres